src/platform/forms-system/src/js/review/ReviewCollapsibleChapter.jsx

```diff
@@ -302,7 +302,8 @@ class ReviewCollapsibleChapter extends React.Component {
         data-chapter={this.props.chapterKey}
       >
         <Element name={`chapter${this.props.chapterKey}ScrollElement`} />
-        <ul className="usa-unstyled-list">
+        {/* eslint-disable-next-line jsx-a11y/no-redundant-roles */}
+        <ul className="usa-unstyled-list" role="list">
```

Comment:
From @joshkimux 's "proposed solution" in the original issue:

> Adding `role="list"` to the `ul` _should_ do the trick, but it didn't work for me in devtools until I explicitly removed the `usa-unstyled-list` class which isn't normal.

That suggests that this solution isn't complete.

Reply:
Ok, did you test it locally? It worked for me and MPS

Reply:
@bkjohnson , `role="list"` is the current, best known fix to this I can find. It doesn't work in all cases, but it's the only safe solution that works in the present. [More in this article.](https://www.scottohara.me/blog/2019/01/12/lists-and-safari.html)

If there are issues with this PR that would outweigh the benefits, we can try and invest in testing this more. Otherwise, I feel this should be OK.

Reply:
@bkjohnson @cvalarida Could you review the PR when you have time? We are blocked until that happens 🙂

Reply:
@kelsonic I did not test locally, I was just going off of the original issue. I was more focused on `usa-unstyled-list` still being there.